src/file_loader.js

```diff
@@ -19,10 +19,14 @@ const readDirSync = dir =>
       ),
       []);
 
```

Comment:
Also would it be possible to create a single array containing all the extensions instead of splitting into two? e.g -
 `const DEFAULT_EXTENSIONS = ['.ts', '.js', '.gql', '.graphql', '.graphqls'];`